src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -125,6 +138,11 @@ private void StartServer()
             Process.BeginOutputReadLine();
         }
 
+        private void Process_Exited(object sender, EventArgs e)
+        {
+            Stop();
```

Comment:
Is it necessary to do anything here except set `Process` to null?

It seems a bit odd to be calling a method called `Stop` when we know the process has already stopped.
If the idea is to put common cleanup code in one place then it might be better to rename the `Stop` method to e.g. `Cleanup` or `EnsureProcessStopped`